Sources/Testing/Running/Configuration.swift

```diff
@@ -258,6 +258,44 @@ public struct Configuration: Sendable {
 
   /// The test case filter to which test cases should be filtered when run.
   public var testCaseFilter: TestCaseFilter = { _, _ in true }
+
+  // MARK: - Expectation checking
+
+  /// Whether or not reflection of values in expressions checked by expectations
+  /// is enabled.
+  ///
+  /// When the value of this property is `false`, value reflection is disabled
```

Comment:
It may be easier for the reader to understand if you express this concept in terms of "when the value is `true`".